x-pack/plugins/spaces/public/lib/constants.ts

```diff
@@ -6,4 +6,6 @@
 
 import chrome from 'ui/chrome';
 
+export const SPACES_FEATURE_DESCRIPTION = `Organize your dashboards and other saved objects into topic-specific spaces.`;
```

Comment:
This description might work a little better because it doesn't use the word "spaces" in the definition.

Organize your dashboards and other saved objects into meaningful categories.